cmd/main.go

```diff
@@ -166,6 +168,30 @@ func setupReconcilersCluster(mgr ctrl.Manager, ramenConfig *ramendrv1alpha1.Rame
 		setupLog.Error(err, "unable to create controller", "controller", "VolumeReplicationGroup")
 		os.Exit(1)
 	}
+
+	fsCGSupport, err := rmnutil.IsFSCGSupport(mgr.GetConfig(), mgr.GetScheme())
+	if err != nil {
+		setupLog.Error(err, "failed to check if ceph fs consistency group is supported")
```

Comment:
In func of `IsFSCGSupport`, it only check the existence of the VolumeGroupSnapshot CRD in the cluster

To prevent the situation where the VolumeGroupSnapshot CRD(it only exists in the new version of external snapshotter) is not installed, which would cause Ramen to continuously crash due to the Informer failing to start because it can't find the VolumeGroupSnapshot CRD. 

Before starting the ReplicationGroupSourceReconciler and ReplicationGroupDestinationReconciler, the existence of the VolumeGroupSnapshot CRD needs to be checked.

that is the reason why it say `ceph fs`, as VolumeGroupSnapshot CRD is only related to the cephfs.

Reply:
If we assume that this version of Ramen will always be installed on an OCP/Kubernetes cluster that includes the VolumeGroupSnapshot CRD, then this code can be removed.

Reply:
I have made the change to return an error in the replicationgroupsource reconciller instead of exiting the process if there is no volumegroupsnapshot CRD installed.

also, if ramenConfig.VolSync.Disabled, the ReplicationGroupDestinationReconciler and ReplicationGroupSourceReconciler will NOT be setup in the latest commit.